packages/contracts/src/components/OwnedByComponent.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0;
+
+import "std-contracts/components/BoolComponent.sol";
+
+uint256 constant ID = uint256(keccak256("ds.component.Loaded"));
+
+///@title OwnedBy for item Character
+///@author Mehdi R.
+///@notice You can deploy this contract as ID for a system
+contract OwnedBy is BoolComponent {
```

Comment:
OwnedBy shouldnt be a boolean, what we want here is the uint256 ID corresponding to a component, so it should be Uint256